### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by rubenperezm**
- Completed the tutorial in a67b48d4cd26f928aea5f21381d7ca66d76d4d6f.
- Adapted the project by removing the APOD panel and adding a left panel for Reana.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
-# reana-jupyterlab-extension
-Reana JupyterLab plugin
+# Reana JupyterLab Extension
+Reana JupyterLab plugin provides a set of tools to interact with the [Reana](https://reanahub.io/) workflow management system from within JupyterLab. 
+
+**This project is currently in development and is subject to change.**
+
+## Requirements
+- JupyterLab >= 4
```
Comment (garciagenrique) [inline README.md:7]: Specify on the README a JLab version >=4 and <5, please.

**Review by garciagenrique — COMMENTED** [review]
Could you please,
 - Remove the bider subfolder for the moment
 - remove the not needed stages of the CI (binder on pr, publish release, etc... we would just like for the moment an install dependencies + build extension).
 
 Thanks !

File: README.md (hunk 1 above)
Comment (rubenperezm) [reply README.md]: Fixed

**Comment by garciagenrique** [comment]
@rubenperezm, if you click on the `Checks` tab, you can navigate until you find the Github action that just run (in this case it failed). An alternative way of getting here is by clicking directly on the `Actions` tab.

Could you please try to understand why the CI is falling ? Thanks

**Comment by rubenperezm** [comment]
It is fixed now. The problem was a double quote (") at the end of the line to run in the `Install Python dependencies` step. There was another error in the `Install JS dependencies step`: 

> YN0028: The lockfile would have been created by this install, which is explicitly forbidden.

It was been fixed by setting `YARN_ENABLE_IMMUTABLE_INSTALLS` to `false`.

**Review by garciagenrique — APPROVED** [review]
lftm ! Thanks @rubenperezm

**Pull request merged**